docs/products/postgresql/reference/log-formats-supported.rst

```diff
@@ -0,0 +1,18 @@
+Log formats supported
+=====================
+
+Aiven for PostgreSQL® supports setting different log formats which are compatible with popular log analysis tools like ``pgbadger`` or ``pganalyze``.
+
+You can use this functionality by navigating to your PostgreSQL® service on the `Aiven console <https://console.aiven.io/>`_.  
+
+Scroll down to the end of the overview page under ``Advanced configuration`` and select the ``Change`` button.  You can then select the ``pg.log_line_prefix`` parameter where there is a set of formats available that can be viewed from the drop down menu:  
```

Comment:
**Advanced configuration** and **Change** are button labels, not parameters, therefore the style should be bold 
```suggestion
Scroll down to the end of the overview page under **Advanced configuration** and select the **Change** button.  You can then select the ``pg.log_line_prefix`` parameter where there is a set of formats available that can be viewed from the drop down menu:  
```